test/integration/node-specific/abstract_cursor.test.ts

```diff
@@ -0,0 +1,114 @@
+import { expect } from 'chai';
+
+import { Collection, MongoAPIError, MongoClient } from '../../../src';
+
+describe('class AbstractCursor', function () {
+  let client: MongoClient;
+
+  let collection: Collection;
+  beforeEach(async function () {
+    client = await this.configuration.newClient().connect();
+
+    collection = client.db('abstract_cursor_integration').collection('test');
+
+    await collection.insertMany(Array.from({ length: 5 }, (_, index) => ({ index })));
+  });
+
+  afterEach(async function () {
+    await collection.deleteMany({});
+    await client.close();
+  });
+
+  context('toArray() with custom transforms', function () {
+    const falseyValues = [0, NaN, '', false];
+    for (const value of falseyValues) {
+      it(`supports mapping to falsey value '${value}'`, async function () {
```

Comment:
Optional, using `inspect(value)` will make 0n print out for bigints otherwise we'll have two tests with the same name